### ISSUE TITLE
[Footer]: Unable to click Privacy and Accessibility in the footer

### ISSUE THREAD

**Issue by jmitch-ibm** [carbon-design-system/carbon-for-ibm-dotcom#11950]
### Description

On [IBM.com](http://ibm.com/) and other IBM sites with this footer, when I have my font size turned down a few steps, when I hover over the Privacy (or Accessibility) links they start jumping around, and I can't click on them.  See attached gif.

![footer-privacy-bug](https://github.com/user-attachments/assets/3008e7fc-9177-406c-8664-fb612133461a)

The same behaviour is visible in the components library: https://www.ibm.com/standards/carbon/web-components/?path=/story/components-footer--default

### Component(s) impacted

Links in the IBM.com Footer

### Browser

Safari

### Carbon for IBM.com version

Current on IBM.com

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

My IBM

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://www.ibm.com/standards/carbon/web-components/?path=/story/components-footer--default

### Steps to reproduce the issue (if applicable)

**(Mac / Safari)**

1. Go to [IBM.com](http://ibm.com/)
2. Reset your font size (View -> Actual Size)
3. Reduce font size two steps (View -> Zoom.  Repeat 2x)
4. Attempt to click Privacy or Accessibility

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
